packages/govuk-frontend-review/src/views/component-preview.njk

```diff
@@ -1,5 +1,9 @@
 {% extends "layouts/" + previewLayout | default('layout') + ".njk" %}
 
+{% set templateTitle = (componentName + ", " + exampleName | lower if exampleName else componentName) | unslugify %}
```

Comment:
Quick one, but `exampleName` falls back to `|| 'default'` so `if exampleName` will always be truthy?

Reply:
Yeah, just checked. Nearly there though!

```
<title>Character count, default - GOV.UK Frontend</title>
```

Reply:
Good idea! 2 tics...